core/server/master/src/main/java/alluxio/master/metastore/InodeStore.java

```diff
@@ -36,19 +38,55 @@
   }
 
   /**
-   * Removes an inode from the inode store. The edge leading to it will also be removed.
+   * Removes an inode from the inode store. This does *not* remove the edge leading to the inode.
```

Comment:
caching inode store wants to operate on inodes and edges separately and by id, not by Inode. So we add id-based methods, and implement the user-friendly wrappers as default implementations.